source/install/self-managed-changelog.md

```diff
@@ -5,13 +5,103 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 Lastest Mattermost Releases:
+- [Release v5.39 - Quality Release](#release-v5-39-quality-release)
 - [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 - [Release v5.36 - Feature Release](#release-v5-36-feature-release)
 - [Release v5.35 - Feature Release](#release-v5-35-feature-release)
-- [Release v5.34 - Feature Release](#release-v5-34-feature-release)
 - [Release v5.31 - ESR](#release-v5-31-esr)
 
+## Release v5.39 - [Quality Release](https://docs.mattermost.com/upgrade/release-definitions.html#quality-release)
```

Comment:
@amyblais - Can you please add a note for the following: Italian, Polish, Korean, and Ukrainian languages have been downgraded to alpha. Korean and Ukrainian are [no longer actively maintained](https://handbook.mattermost.com/contributors/join-us/localization#translation-quality).

Reply:
@cwarnermm I added a note. However, Italian and Polish are still marked as "Beta" in Account Settings / System Console, so not sure if it will cause confusion.

Reply:
Thanks, @amyblais - You're right, that would cause confusion if left as-is. If a PR was submitted today to update Italian and Polish in Account Settings to "Alpha", would that be acceptable on the day of code freeze?

Reply:
@cwarnermm 
> If a PR was submitted today to update Italian and Polish in Account Settings to "Alpha", would that be acceptable on the day of code freeze?

That would be fine.

Reply:
Excellent! https://github.com/mattermost/mattermost-webapp/pull/8818 addresses this.